src/eureka/S3_data_reduction/nirspec.py

```diff
@@ -155,3 +155,201 @@ def fit_bg(dataim, datamask, n, meta, isplots=0):
         The current integration number.
     """
     return nircam.fit_bg(dataim, datamask, n, meta, isplots=isplots)
+
+
+def find_column_coms_and_shifts(data):
+    '''
+    Takes a data array (int, rows, columns) and find the 
+    center of mass (COM) in pixels for each column. It then returns the needed
+    shift to apply to each column to bring the COM to the center
+
+    Input:
```

Comment:
Please follow the exact same documentation style used by the other functions - this is required for the documentation to all automatically compile nicely onto the website.

Reply:
should be updated now!